### ISSUE TITLE
Crash in LubTypeVariableAnnotator.annotateEmptyLub

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#421]
```
This crash occurs with the latest version of the checker (I tested against the 
artifacts from this build: 
http://tern.cs.washington.edu:8080/job/checker-framework/jdk=JDK_7/245/)


```

Original issue reported on code.google.com by `cus...@google.com` on 25 Mar 2015 at 1:22

Attachments:
- [stacktrace.txt](https://storage.googleapis.com/google-code-attachments/checker-framework/issue-421/comment-0/stacktrace.txt)

**Comment by smillst** [comment]
Fixed in b0b56a5865917c785875a19dc54b8a25bcf3babd

**Comment by smillst** [comment]
Test case checker/tests/nullness/generics/Issue421.java
